### PR TITLE
Resolves issues #84 #97

### PR DESCRIPTION
**Pull request by jwdonahue**

### CODE REVIEW

File: CryptoNet.Cli/ExampleAes.cs
Comment (jwdonahue) [inline CryptoNet.Cli/ExampleAes.cs:1]: Woops. Accidently edited this file at some point. No matter, we'll be removing it, once I am sure we've got equivalent coverage in the unit tests.

File: CryptoNet.UnitTests/GlobalSuppressions.cs
Comment (jwdonahue) [inline CryptoNet.UnitTests/GlobalSuppressions.cs:1]: These suppressions were not needed.

File: CryptoNet.vpj
Comment (maythamfahmi) [inline CryptoNet.vpj:1]: suppose this file to be gitignored? 

https://github.com/maythamfahmi/CryptoNet/issues/97#issuecomment-2421878297

File: CryptoNet.vpw
Comment (maythamfahmi) [inline CryptoNet.vpw:1]: suppose this file to be gitignored? 

https://github.com/maythamfahmi/CryptoNet/issues/97#issuecomment-2421878297

File: CryptoNet.vpwhist
Comment (maythamfahmi) [inline CryptoNet.vpwhist:1]: suppose this file to be gitignored? 

https://github.com/maythamfahmi/CryptoNet/issues/97#issuecomment-2421878297

File: CryptoNet.vtg
Comment (maythamfahmi) [inline CryptoNet.vtg:1]: suppose this file to be gitignored? 

https://github.com/maythamfahmi/CryptoNet/issues/97#issuecomment-2421878297

File: CryptoNet.UnitTests/CryptoNet.UnitTests.csproj
Comment (maythamfahmi) [inline CryptoNet.UnitTests/CryptoNet.UnitTests.csproj:1]: add this between line 41 and 42

<None Update="Resources\RsaKeys\publicKey.pub">
    <CopyToOutputDirectory>Always</CopyToOutputDirectory>
</None>
<None Update="Resources\RsaKeys\privateKey">
    <CopyToOutputDirectory>Always</CopyToOutputDirectory>
</None>

Comment (jwdonahue) [reply CryptoNet.UnitTests/CryptoNet.UnitTests.csproj:1]: Done.

**Review by maythamfahmi — APPROVED** [review]

**Review by maythamfahmi — APPROVED** [review]

**Pull request merged**